tests/rng/test.c

```diff
@@ -154,7 +154,7 @@ void test_distributions(uint32_t samples)
 
         /* count bits */
         for (int i = 0; i < 32; i++) {
-            if (value & (1 << i)) {
+            if (value & (1u << i)) {
```

Comment:
Shouldn't it be `1LU` to fit 32-bit?

Reply:
Or even better `UINT32_C(1)` for all architectures indeed. I will update.